php/ext/google/protobuf/protobuf.h

```diff
@@ -43,6 +43,24 @@ const zval *get_generated_pool();
 #define GC_DELREF(h) --GC_REFCOUNT(h)
 #endif
 
+#if PHP_VERSION_ID < 80000
```

Comment:
Please add a link to this, to help explain these changes:
https://github.com/php/php-src/blob/898bb97706326311a367aaef35b3f95510100f3c/UPGRADING.INTERNALS#L37-L39

Reply:
done